src/TorchVision/AutoAugment.cs

```diff
@@ -0,0 +1,162 @@
+        {
+            if (op_name == "ShearX") {
+                img = F.affine(
+                    img,
+                    angle: 0.0f,
+                    translate: new[] { 0, 0 },
+                    scale: 1.0f,
+                    shear: new[] { (float)((180.0f / Math.PI) * Math.Atan(magnitude)), 0.0f },
+                    interpolation: interpolation,
+                    fill: fill?.FirstOrDefault());
+            } else if (op_name == "ShearY") {
+                img = F.affine(
+                    img,
+                    angle: 0.0f,
+                    translate: new[] { 0, 0 },
+                    scale: 1.0f,
+                    shear: new[] { 0.0f, (float)((180.0f / Math.PI) * Math.Atan(magnitude)) },
+                    interpolation: interpolation,
+                    fill: fill?.FirstOrDefault());
+            } else if (op_name == "TranslateX") {
+                img = F.affine(
+                    img,
+                    angle: 0.0f,
+                    translate: new[] { (int)(magnitude), 0 },
+                    scale: 1.0f,
+                    interpolation: interpolation,
+                    shear: new[] { 0.0f, 0.0f },
+                    fill: fill?.FirstOrDefault());
+            } else if (op_name == "TranslateY") {
+                img = F.affine(
+                    img,
+                    angle: 0.0f,
+                    translate: new[] { 0, (int)(magnitude) },
+                    scale: 1.0f,
+                    interpolation: interpolation,
+                    shear: new[] { 0.0f, 0.0f },
+                    fill: fill?.FirstOrDefault());
+            } else if (op_name == "Rotate") {
+                img = F.rotate(img, magnitude, interpolation, fill: fill);
+            } else if (op_name == "Brightness") {
+                img = F.adjust_brightness(img, 1.0 + magnitude);
+            } else if (op_name == "Color") {
+                img = F.adjust_saturation(img, 1.0 + magnitude);
+            } else if (op_name == "Contrast") {
+                img = F.adjust_contrast(img, 1.0 + magnitude);
+            } else if (op_name == "Sharpness") {
+                img = F.adjust_sharpness(img, 1.0 + magnitude);
+            } else if (op_name == "Posterize") {
+                img = F.posterize(img, (int)magnitude);
+            } else if (op_name == "Solarize") {
+                img = F.solarize(img, magnitude);
+            } else if (op_name == "AutoContrast") {
+                img = F.autocontrast(img);
+            } else if (op_name == "Equalize") {
+                img = F.equalize(img);
+            } else if (op_name == "Invert") {
+                img = F.invert(img);
+            } else if (op_name == "Identity") {
+                // Pass
+            } else {
+                throw new ArgumentException($"The provided operator {op_name} is not recognized.");
+            }
+
+            return img;
+        }
+
+
+        internal class RandAugment : ITransform
```

Comment:
I would place the class in the 'Modules' namespace under 'TorchSharp,' which is the pattern we (have tried to) use elsewhere.

Reply:
I don't think it needs to be exposed under the modules namespace, its `internal`. Other transforms see [Rotate.cs ](https://github.com/dotnet/TorchSharp/blob/main/src/TorchVision/Rotate.cs) follow the pattern I've used here.

Reply:
It's easier to find when you're cross-referencing code in VS. 

Not a biggie, but consistency (if you find places where the current code is not) is helpful across a code base.

Reply:
@NiklasGustafsson I think all files under `TorchVision/*.cs` currently! I feel its a separate refactoring issue.